n_publishers: 1000 # [-]

timer_rate: 100.0 # [Hz]
